core/host_api/impl/crypto_extension.cpp

```diff
@@ -204,6 +204,17 @@ namespace kagome::host_api {
 
       auto res = self->sr25519_provider_->verify(signature, msg, pubkey);
       bool is_succeeded = res && res.value();
+      if (not is_succeeded) {
+        SL_DEBUG(self->logger_,
```

Comment:
Hm... Not sure if it good idea to use multiline log messages

Reply:
It's a debug. And it's real pain to copy these huge addresses and messages from one liner. Try to make dev live easier.

Reply:
If it's more preferred I can make 4 separate log messages. Not a big deal.

Reply:
Sure, it is not a big deal. But it is best practice to use uniform ways for similar situations, and this case is not exception.